modules/visual_script/doc_classes/VisualScriptLists.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="VisualScriptLists" inherits="VisualScriptNode" category="Core" version="3.2">
+	<brief_description>
+		A Visual Script virtual class for in-graph editable nodes.
+	</brief_description>
+	<description>
+		A Visual Script virtual class that defines the shape and the default behaviour of the nodes that have to be in-graph editable nodes.
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="add_input_data_port">
+			<return type="void">
+			</return>
+			<argument index="0" name="arg0" type="int" enum="Variant.Type">
```

Comment:
You forgot to name arguments in the bindings.

Reply:
Thanks for the catch. :)